plugins/ModuleLevelTrigger.hpp

```diff
@@ -224,6 +224,13 @@ class ModuleLevelTrigger : public dunedaq::appfwk::DAQModule
   std::atomic<metric_counter_type> m_lc_kLive{ 0 };
   std::atomic<metric_counter_type> m_lc_kPaused{ 0 };
   std::atomic<metric_counter_type> m_lc_kDead{ 0 };
+  std::atomic<uint64_t>            m_tc_data_vs_system;
+  std::atomic<uint64_t>            m_td_made_vs_ro;
+  std::atomic<uint64_t>            m_td_send_vs_ro;
+
+  // Latency
+  std::atomic<bool>                m_first_tc;
+  std::atomic<uint64_t>            m_initial_offset;
```

Comment:
I don't quite understand why these need to be atomic? MLT has only one thread, `m_send_trigger_decisions_thread`, so what else do we need to synchronise at the time measurement point? Atomics are normally used when memory is shared between multiple running threads, but only one thread is using these?

Reply:
I was wrong here, resolved. I now see there are at least two threads using these.